### DISCUSSION TITLE
Welcome to CyberConnectApp Discussions!

### DISCUSSION THREAD

**Discussion by wonderBH717** [wonderBH717/CyberConnectApp#5] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wonderBH717** [comment, marked as answer]
I got a issue for connecting cyber connect.

**Reply by bh0717-dev** [reply]
Please don't use original cyberconnect api.
You have to implement it.

**Comment by wonderBH717** [comment]
I can't access social network with cyber connect api

**Reply by bh0717-dev** [reply]
You can't access social network with cyberconnect api.
Becuase Cyberconnect api doesn't support social api yet.
